internal/yunikorn/sync_int_test.go

```diff
@@ -64,22 +64,22 @@ func TestClient_sync_Integration(t *testing.T) {
 
 	err = s.sync(ctx)
 	if err != nil {
-		t.Errorf("error starting up client: %v", err)
+		t.Fatalf("error starting up client: %v", err)
 	}
 
 	assert.Eventually(t, func() bool {
 		partitions, err := repo.GetAllPartitions(ctx)
 		if err != nil {
-			t.Fatalf("error getting partitions: %v", err)
+			t.Logf("error getting partitions: %v", err)
 		}
-		return len(partitions) > 0
-	}, 10*time.Second, 250*time.Millisecond)
+		return err == nil && len(partitions) > 0
+	}, 15*time.Second, 500*time.Millisecond)
 
 	assert.Eventually(t, func() bool {
 		history, err := repo.GetApplicationsHistory(ctx)
 		if err != nil {
-			t.Fatalf("error getting applications history: %v", err)
+			t.Logf("error getting applications history: %v", err)
 		}
-		return len(history) > 0
-	}, 10*time.Second, 250*time.Millisecond)
+		return err == nil && len(history) > 0
+	}, 15*time.Second, 500*time.Millisecond)
```

Comment:
I've never had this pass, even when changing the timeout to 2 minutes.  
Where in the test do we submit application jobs? I do not see it.

Reply:
Update: it does pass when I set the timeout to 300 seconds (running on my work Macbook).